tests/integration/targets/object_cluster_list/tasks/main.yaml

```diff
@@ -10,18 +10,6 @@
 #    - assert:
 #        that:
 #          - no_filter.clusters | length >= 1
-#
-#    - name: List regions with filter on region
```

Comment:
This no longer functions in both the old and new module implementations. Given this is a deprecated API, do we think it's worth keeping and investigating further?

Reply:
I think we can skip and save effort since it's deprecated.

Reply:
Sounds good to me!